### PR TITLE
chore: add favicons

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/open-feature/docs.openfeature.dev/issues/233

### CODE REVIEW

**Comment by Arhell** [comment]
cc @beeme1mr

**Comment by Arhell** [comment]
https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-235--lucky-creponne-baf9b3.netlify.app%2F#.Y66qsnbP23A

File: static/favicons/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```
Comment (beeme1mr) [inline static/favicons/site.webmanifest:2]:
```suggestion
    "name": "OpenFeature",
```

File: static/favicons/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "short_name": "",
```
Comment (beeme1mr) [inline static/favicons/site.webmanifest:3]:
```suggestion
    "short_name": "OpenFeature",
```

File: static/favicons/site.webmanifest (hunk 1 above)
Comment (Arhell) [reply static/favicons/site.webmanifest:2]: ok, thanks

File: static/favicons/site.webmanifest (hunk 2 above)
Comment (Arhell) [reply static/favicons/site.webmanifest:3]: ok, thanks

**Comment by Arhell** [comment]
@beeme1mr  done

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**